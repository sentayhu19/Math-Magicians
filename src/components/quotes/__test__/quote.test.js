import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Quote from '../quotes';

describe('Quote page of the App', () => {
  test('Builds the snapchot of the Quote component safely', () => {
    const rend = renderer.create(<Quote />);
    expect(rend.toJSON()).toMatchSnapshot();
  });
  test('Render Quote component', () => {
    render(<Quote />);
  });
});
