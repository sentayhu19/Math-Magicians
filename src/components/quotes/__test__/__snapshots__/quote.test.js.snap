// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote page of the App Builds the snapchot of the Quote component safely 1`] = `
<p
  className="quote"
>
  mathematician is not about numbers, equations, computations,
  <br />
  or algorithms: it is about understanding -William paul Thurston
</p>
`;
